ludwig/models/llm.py

```diff
@@ -178,6 +186,9 @@ def get_input_ids(
         """Returns the input ids for the text feature input."""
         return inputs[self.config_obj.input_features[0].name].type(torch.int32)
 
+    ###SXK just a thought here, based on the current use cases does it make sense to have a BaseLLM
+    ###class that we can build custom LLM classes, maybe the base LLM class contains a generic implementation
+    ###of the forward method, lets discuss this some more if it makes sense
```

Comment:
Hmm, we could, but curious where/how you think that might be useful. I don't see why not though